BackofficeBundle/Resources/public/less/editTable.less

```diff
@@ -1,3 +1,11 @@
+.ribbon-form-button {
```

Comment:
you know i add the css of actions buttons in this file because i found the .back-to-list css there